tutorials/custom_evaluator/director/README.md

```diff
@@ -20,6 +18,15 @@ docker push $REGISTRY/custom-eval-tutorial-director
 ```
 
 Step4: Update the install yaml for your setup.
-```
-sed "s|REGISTRY_PLACEHOLDER|$REGISTRY|g" director.yaml | kubectl apply -f -
-```
+
+- For GKE users, run:
+    ```bash
+    sed "s|REGISTRY_PLACEHOLDER|$REGISTRY|g" director.yaml | kubectl apply -f -
+    ```
+- For Minikube users, run:
```

Comment:
Minor change:

For Minikube, [use local images](http..) by running the following command:

```bash
..
```